ckanext/datajson/datajson.py

```diff
@@ -396,8 +396,11 @@ def is_part_of_to_package_id(self, ipo, harvest_object):
             Only search for datasets that are the parent of a collection.
             """
         ps = p.toolkit.get_action('package_search')
-        query = 'extras_identifier:"{}" AND extras_collection_metadata:true'.format(ipo)
-        results = ps(self.context(), {"fq": query})
+        query = 'extras_identifier:{} AND extras_collection_metadata:true'.format(ipo)
```

Comment:
This must have been an old branch, you dropped/lost my recent change quoting the extras_identifier: `"{}"`

Reply:
I probably copied badly.  But yes, you're right.  Thanks for catching this 😅